CONTRIBUTING.md

```diff
@@ -281,11 +318,11 @@ Here's a test method following those conventions:
     }
 ```
 
-### Test Utility Methods
+#### Test Utility Methods
 
-Because we sometimes need to reuse some setup or assertions in our tests, we tend to write utility methods. Those methods should be private (or internal in a dedicated class/file if they need to be shared across tests).
+Because we sometimes need to reuse some setup or assertions in our tests, we tend to write utility methods. 
+Those methods should be private (or internal in a dedicated class/file if they need to be shared across tests).
 
-- `fun mockSomething([args]): T`: methods setting up a mock. These methods must return the mocked instance;
```

Comment:
why do we remove this line?

Reply:
As per our discussions, mocks are only listeners that shouldn't need any configuration. Only stubs should require configuration, so this line isn't relevant anymore